incubator/elasticsearch/templates/coordinating-svc.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "coordinating.fullname" . }}"
+  labels:
+    app: "{{ template "name" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  type: "{{ .Values.coordinating.service.type }}"
+  ports:
+  - name: http
+    port: {{ .Values.coordinating.service.port }}
+    targetPort: http
+  selector:
+    app: "{{ template "name" . }}"
+    release: "{{ .Release.Name }}"
+    role: "coordinating-only"
```

Comment:
why not use the value `role: coordinating` instead of `coordinating-only`?

Reply:
It's metadata and it's more descriptive but I'm open to change it.

Reply:
don't have a strong opinion either